Formula/php.rb

```diff
@@ -70,6 +71,9 @@ def install
     ENV.append "CPPFLAGS", "-DU_USING_ICU_NAMESPACE=1"
 
     config_path = etc/"php/#{php_version}"
+    # Prevent system pear config from inhibiting pear install
+    (config_path/"pear.conf").delete if (config_path/"pear.conf").exist?
```

Comment:
I'd be wary of deleting user files here. Additionally, bear in mind this won't work for bottled installs.

Reply:
I am a bit wary, but pear has a two-tier configuration system that provides a "system" config and a "user" config. Users _should not_ be modifying the system config and the post_install will rebuild it. The guard is there specifically for source builds, as anyone that has this config file with configuration pointed at a pear repository where the pear files are already installed will skip the pear installation of binaries in the cellar. The would mean build from source upgrades would be missing the `pear` and `pecl` binaries.